frontend/views.py

```diff
@@ -1,5 +1,6 @@
+from django.http.request import HttpRequest
 from django.shortcuts import render
 
-# Create your views here.
-def index(request):
-    return render(request, 'frontend/index.html')
\ No newline at end of file
+
+def index(request: HttpRequest):
```

Comment:
I don't think this needs a specific type, but I'm curious to hear your thoughts

Reply:
It doesn't but it allows vscode to add better autocompletion support

Reply:
In general I've noticed that typing the variables results in better autocompletion. Variable declarations get typed a bit differently so sometimes you might see `... # type: <some-type>` after a declaration which lets the parser know the type of the variable

Reply:
Good tip!